eng/common/pipelines/templates/steps/verify-changelogs.yml

```
@@ -6,6 +6,12 @@ parameters:
   default: succeeded()
 
 steps:
+  - pwsh: |
+      Get-ChildItem -Recurse -Path "$(Build.SourcesDirectory)/sdk" -Include ci.yml `
+        | % { Get-Content $_ | yq -o=json > "$($_.FullName).json" }
```

```suggestion
        | ForEach-Object { Get-Content $_ | yq -o=json > "$($_.FullName).json" }
```

As a convention we try to use the full name for pipeline functions in scripts.

---

Also I've found `-Raw` is good when passing gci to convert from json, etc. so it doesn't pass things as newline separated arrays around.

---

Unless that's the intent here...

---

> As a convention we try to use the full name for pipeline functions in scripts.

Yeah and I usually remember to replace it from local script invocation.